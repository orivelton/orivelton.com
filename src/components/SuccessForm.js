import React from 'react';
import { Link } from 'gatsby';

const SuccessForm = () => (
  <p>
    <strong>
      Thanks for sending me a message! I'll send you a response soon.
    </strong>
    <Link to="/blog" className="nav__link" activeClassName='nav__link--active' title="Got to Blog page">Let's see the Blog?!!</Link>
  </p>
);

export default SuccessForm;