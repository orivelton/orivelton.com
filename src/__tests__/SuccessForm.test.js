import React from 'react';
import renderer from 'react-test-renderer';
import SuccessForm from '../components/SuccessForm';

describe('src/components/SuccessForm.js', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<SuccessForm />)
      .toJSON()
    expect(tree).toMatchSnapshot();
  });
});
