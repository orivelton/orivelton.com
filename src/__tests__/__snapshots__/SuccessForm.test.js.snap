// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/SuccessForm.js renders correctly 1`] = `
<p>
  <strong>
    Thanks for sending me a message! I'll send you a response soon.
  </strong>
  <a
    className="nav__link"
    href="/blog"
    title="Got to Blog page"
  >
    Let's see the Blog?!!
  </a>
</p>
`;
